tools/privacyidea-migrate-linotp.py

```diff
@@ -120,6 +194,14 @@ def migrate():
                         Column("sync_window", Integer(), default=1000),
                         Column("rollout_state", Unicode(10), default=u''))
 
+    tokenowner_table = Table("tokenowner", metadata,
```

Comment:
Could we use the table definitions from `models.py`? This way we always have to keep track of changes in the tables.

Also do we need the `resolver_config_table` and `resolverrealm_table`? They aren't used anywhere.

Reply:
You are basically right.
But there was a reason, why we can not use it. I think because they are defined in a different way - not via object mapping definition.
And as we want to run the script out of the application scope (for performance reason) we need to define it in a different way.

Reply:
We could use
```python
from privacyidea.models import Token
token_table = Token.__table__
```
but i'll save that for when we have tests for the tools :-)